### DISCUSSION TITLE
bind or store paragraph control values in a sql server table

### DISCUSSION THREAD

**Discussion by AK-Bharath** [Budibase/budibase#10704] [Help] [answered]
Hi There,

I am trying to create a survey application in budibase.
I am using form block to create questions for this survey application.
I would like to store all questions and answers in SQL server table.

I am loading all the questions in paragraph control in budibase.
while saving the questions and answers in database,  I would like to save both questions and answers in sql server table.
I am not able to save questions which are in paragraph (not able to bind or save paragraph text to database table)

Please advise how to store the paragraphs values to a SQL table in form.

Any help is highly appreciated.

With many thanks,

Bharath AK

**Comment by melohagan** [comment, marked as answer]
Hi @AK-Bharath 

The paragraph component is designed to display data. It is not intended for data collection. 

Please have a look at the docs for form inputs: https://docs.budibase.com/docs/text-inputs

**Reply by AK-Bharath** [reply]
Hi Melohagan,

Apologies for delayed response and thanks for your email.
Is it possible to achieve this by building some components, if it is possible, give some ideas or overview, how build this requirement into a component.

with many thanks,

Bharath AK

**Reply by melohagan** [reply]
You can read more about writing custom components here: https://docs.budibase.com/docs/custom-component

**Comment by AK-Bharath** [comment]
Thank you

Kind Regards,
Bharath AK
